decidim-results/app/models/decidim/results/result.rb

```diff
@@ -19,7 +19,7 @@ def commentable?
 
       # Public: Overrides the `accepts_new_comments?` Commentable concern method.
       def accepts_new_comments?
-         commentable? && !feature.active_step_settings.comments_blocked
+        commentable? && !feature.active_step_settings.comments_blocked
```

Comment:
I don't think so Oriol, we don't have interfaces for settings so this just happens to be a convention. If we unified comment setting somehow I'd be in. Otherwise I'd explicitly implement the method.